doc/api/http.md

```diff
@@ -564,6 +573,29 @@ Once a socket is assigned to this request and is connected
 
 Returns `request`.
 
+### request.socket
+<!-- YAML
+added: v0.3.0
+-->
+
+* {net.Socket}
+
+Reference to the underlying socket. Usually users will not want to access
+this property. In particular, the socket will not emit `'readable'` events
+because of how the protocol parser attaches to the socket. After
+`response.end()`, the property is nulled. The `socket` may also be accessed
+via `request.connection`.
+
+Example:
+
+```js
+const http = require('http');
+http.createServer((req, res) => {
+  const clientIp = req.socket.remoteAddress;
+  res.end(`Your IP is ${clientIp}`);
```

Comment:
Sorry to nitpick, but "IP adress" would be more correct, as IP is just the protocol. Also, I'd add a second print for the port. Same goes for the `res` example below.